libsplinter/src/rest_api/sessions/mod.rs

```diff
@@ -78,7 +78,7 @@ where
     extra_validation(claims)
 }
 
-fn default_validation(issuer: &str) -> Validation {
+pub(crate) fn default_validation(issuer: &str) -> Validation {
```

Comment:
Should this be restricted to biome?

Reply:
We would have to move them into the biome crate, and I think the sessions stuff should stay together in the rest-api module.